.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I see that Istio lately moved PR to a form (https://github.com/istio/istio/commit/7f810ef1b1c84e6e0d5442d60aaaf5d4c14e6d26)
maybe we should also do it? 
you can do validations for required fields and it looks better I think